cmd/kubernetes_version.go

```diff
@@ -69,6 +72,30 @@ func extractVersion(data []byte) (string, error) {
 	return ver, nil
 }
 
+func getWebDataRetry(srvURLs []string, token string, cacert *tls.Certificate, timeoutDuration, waitDuration time.Duration) (data []byte, err error) {
+	timeout := time.After(timeoutDuration)
+	wait := time.Duration(waitDuration)
+	for {
+		select {
+		case <-timeout:
+			err = errors.New("timeout - getting Kubernetes version using REST API")
+			return
+		default:
+			for _, srvURL := range srvURLs {
```

Comment:
Per comment below, I don't think this should be a list. It should either be set by env vars, or it should be the default value.